.gitlab-ci.yml

```diff
@@ -79,18 +76,24 @@ build:rust:linux:release:          &build
     - web
   tags:
     - linux-docker
+  before_script:
+   - ./scripts/build.sh
   script:
     - time cargo build --release --verbose;
-    - ./target/release/substrate --version
-    - sha256sum ./target/release/substrate
-
+    - mkdir -p ./artifacts
+    - mv ./target/release/substrate ./artifacts/.
```

Comment:
the `target/release` contains also a folder `completion-scripts` with a bunch of scripts build for bash-completion of the CLI. Are we not shipping them also?

Reply:
I assume that if we want to ship them, we would require an installation script? As these completions would probably need to be installed somewhere in the system?